website/documentation/guides/kubernetes-hardening/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Kubernetes Cluster Hardening Procedure for Cluster Admins
+description: "Using Cloud Access Management (CAM) for compliant user management of your Gardener Projects"
+level: advanced
+category: Security
+persona: Operators
+publishdate: 2023-10-10
+tags: ["task"]
+---
+
+## Overview
+The Gardener team takes security very seriously, which is why we mandate the Security Technical Implementation Guide (STIG) for Kubernetes as published by the Defense Information Systems Agency (DISA) [here](https://public.cyber.mil/stigs/downloads/). Thus, Gardener has decided and aligned with SGS to adopt and show SCR03 compliance with the DISA Kubernetes STIGs. In combination with the control change above, this offers Gardener stakeholders the opportunity to adopt and show SCR03 compliance with DISA Kubernetes STIG as well. The latest release in machine readable format can be found in the [STIGs Document Library](https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=container-platform) by searching for Kubernetes.
```

Comment:
> Thus, Gardener has decided and aligned with SGS to adopt and show SCR03 compliance with the DISA Kubernetes STIGs.

This sentence should be deleted. We shall not mention SGS in the public site (at least not as of today).
We shall rethink the whole paragraph.

Reply:
On top of that, probably as additional public page, we shall list all rules, which are responsibility to Service Provider (this can be a 2nd step)